Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -98,11 +98,19 @@
                     </MenuFlyoutItem.Icon>
                 </MenuFlyoutItem>
                 <MenuFlyoutItem
-                    x:Uid="StatusBarControlGridViewSmall"
-                    Command="{x:Bind AppSettings.ToggleLayoutModeGridViewSmall}"
-                    Text="Grid View (Small)">
```

Comment:
It looks like you removed this layout option by mistake.

Reply:
I fixed that issue